### ISSUE TITLE
Remove all the incomplete model loaders from core

### ISSUE THREAD

**Issue by riccardobl** [MeFisto94/jme-discussion#9]
I think we should remove all the incomplete/broken model loaders from the core.
I would keep only gltf and obj.
Having so many choices confuses the newcomers, increases the code we need to maintain... and they don't fully work anyway.

**Comment by MeFisto94** [comment]
Agreed. In the best case every loader would be it's own module and especially push those formats towards tools to generate j3os. That doesn't mean you can't load .blend if it drives your gears, but it doesn't have to be included everywhere and advertised as such.
